File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/02.Conditional-Statements/02-exercise/08.lunchBreak.js
function lunchBreak(input) {

    const series = input[0];
    const durationEpisode = Number(input[1]);
    const durationRest = Number(input[2]);

    const lunchTime = durationRest / 8;
    const relaxationTime = durationRest / 4;

    const freeTime = durationRest - lunchTime - relaxationTime;
    const difference = Math.abs(freeTime - durationEpisode);

    if (freeTime >= durationEpisode) {
        console.log(`You have enough time to watch ${series} and left with ${Math.ceil(difference)} minutes free time.`);
    } else {
        console.log(`You don't have enough time to watch ${series}, you need ${Math.ceil(difference)} more minutes.`);
    }

}
lunchBreak(["Game of Thrones", "60", "96"]);

File: Python_Basics/Homeworks-And-Labs/02.conditional_statements/02.exercise/08.lunch_break.py
from math import ceil

series = input()
durationEpisode = int(input())
durationRest = int(input())

lunchTime = durationRest / 8
relaxationTime = durationRest / 4

freeTime = durationRest - lunchTime - relaxationTime
difference = abs(freeTime - durationEpisode)

if freeTime >= durationEpisode:
    print(f'You have enough time to watch {series} and left with {ceil(difference)} minutes free time.')
else:
    print(f"You don't have enough time to watch {series}, you need {ceil(difference)} more minutes.")